src/status_im2/contexts/wallet/events.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im2.contexts.wallet.events
+  (:require [utils.re-frame :as rf]
+            [taoensso.timbre :as log]))
+
+(rf/defn get-wallet-tokens
+  {:events [:wallet-2/get-wallet-tokens]}
+  [_ accounts]
+  {:json-rpc/call [{:method     "wallet_getWalletToken"
+                    :params     [(map :address accounts)]
+                    :on-success #(rf/dispatch [:wallet-2/get-wallet-tokens-success %])
+                    :on-error   #(log/info "failed " %)}]})
```

Comment:
Not for this PR, but I think we generally need to have more error handling in the app regarding feedback to user when something goes wrong
cc @flexsurfer @J-Son89 @ilmotta

Reply:
yep, I agree. The designs should incorporate this imo so if the error states are lacking then it's something we should discuss with the designers

Reply:
We could probably add some prefix tag in the log string so when we see the logs we know which event failed. Maybe something like `#(log/info "[:wallet-2/get-wallet-tokens] failed: " %)`

Reply:
Yes, this has been raised up a couple times. As @J-Son89 said, the error flows should be part of the designs.

Reply:
Done